src/audacity.desktop.in

```diff
@@ -44,6 +50,12 @@ Comment[tr]=Ses dosyalarını kaydetme ve düzenleme
 Comment[uk]=Запис і редагування звукових файлів
 Comment[zh_CN]=录音和编辑音频文件
 Comment[zh_TW]=錄音和編輯音訊檔案
+# keywords added
+
+Keywords= Audio 
+Keywords= Recording
+Keywords= Editing 
```

Comment:
You're only allowed to have one `Keywords`. The corresponding bug already showed a syntax: 
```Keywords=audio;editing;recorder;[...];```